citations-tool/tool/src/webapp/vm/citation/_listNestedCitations.vm

```diff
@@ -39,6 +39,24 @@
 		#set( $addH1SubsectionButtonId = 'addH1SubsectionButton' + $nestedSection.getLocation() )
 		#set( $buttonsDiv = 'buttonsDiv' + $nestedSection.getLocation() )
 		#set( $num = $num + 1)
+		#set( $citationNo = 0)
```

Comment:
If you don't want to pre-calculate it make the Java method accessible in the context and then just call that. Templates really should be simple and just concerned with displaying the data, not doing calculations.